fesvr/elfloader.cc

```diff
@@ -32,6 +34,12 @@ std::map<std::string, uint64_t> load_elf(const char* fn, memif_t* memif, reg_t*
   assert(size >= sizeof(Elf64_Ehdr));
   const Elf64_Ehdr* eh64 = (const Elf64_Ehdr*)buf;
   assert(IS_ELF32(*eh64) || IS_ELF64(*eh64));
+  unsigned xlen = IS_ELF32(*eh64)?32:64;
+  if(required_xlen != 0){
+    if(required_xlen != xlen){
```

Comment:
Please format code consistent with the rest of Spike: spaces around binary or ternary operators, space after `if` and before open bracket.

No need for nested `if` here, this is simply `&&`.